backend/hitas/views/ownership.py

```diff
@@ -25,12 +26,18 @@ class Meta:
 
 
 class OwnershipSerializer(serializers.ModelSerializer):
+
+    id = serializers.SerializerMethodField()
     owner = OwnerSerializer()
     percentage = HitasDecimalField(required=True)
 
+    def get_id(self, instance: ExternalHitasModel) -> str:
+        return instance.uuid.hex
+
     class Meta:
         model = Ownership
         fields = [
+            "id",
```

Comment:
`id` can be removed from here as well :)

Reply:
Additional ownerships option on condition of sale creation needs this, see [here](https://github.com/City-of-Helsinki/hitas/pull/212/files#diff-371f597cf9397b67f5f3749ab0f5c08c290e53cf39589034af23ead33edee12bR79).